### PR TITLE
Update issue templates - Closes #1615

### LINKED ISSUE #1615: Update issue templates
### Description
The issue templates need to be updated according to https://github.com/LiskHQ/discussions/discussions/47

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1615

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -1,14 +1,29 @@
...
 labels: 'type: feature'
 assignees: ''
 
 ---
 
-### Is your feature request related to a problem?
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+### Motivation
+Why is this new content needed?
+What is the intention/ motivation of creating it?
```
Comment (chris529) [inline .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
What is the intention/motivation for creating it?
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -1,14 +1,29 @@
...
 
-### Description
-A clear and concise description of the feature you want.
+### Content structure
+Describe the desired structure of the new content.
+Create a TOC as bullet list, and add more information about the desired content as possible:
```
Comment (chris529) [inline .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
Create a TOC in the form of a bullet-point list, and add as much information as possible regarding the desired content:
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -1,14 +1,29 @@
...
+- mention important information to include or emphasize on
+- leave additional comments or suggestions, e.g. for creating diagrams
+
+### Location
+
+#### Affected branch / URL
+Choose the branch that should be used as base branch for this issue.
```
Comment (chris529) [inline .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
Choose the applicable branch that should be used as the base branch for this issue.
```

File: .github/ISSUE_TEMPLATE/improvement.md [hunk 4]
```diff
@@ -1,10 +1,25 @@
...
-about: Suggest an improvement idea for the Lisk documentation
+about: Used, when improvements to existing content are suggested.
 title: ''
 labels: 'type: improvement'
 assignees: ''
 
 ---
 
+### Motivation
+Why  does this content need improvement?
```
Comment (chris529) [inline .github/ISSUE_TEMPLATE/improvement.md]:
```suggestion
Why does this content need improvement?
```

File: .github/ISSUE_TEMPLATE/improvement.md [hunk 5]
```diff
@@ -1,10 +1,25 @@
...
+
 ### Description
+Explain, how the content should be improved.
```
Comment (chris529) [inline .github/ISSUE_TEMPLATE/improvement.md]:
```suggestion
Explain how the content should be improved.
```

File: .github/ISSUE_TEMPLATE/improvement.md [hunk 6]
```diff
@@ -1,10 +1,25 @@
...
+
+### Location
+
+#### Affected branch / URL
+Choose the branch that should be used as base branch for this issue.
```
Comment (chris529) [inline .github/ISSUE_TEMPLATE/improvement.md]:
```suggestion
Choose the applicable branch that should be used as the base branch for this issue.
```

File: .github/ISSUE_TEMPLATE/update-content.md [hunk 7]
```diff
@@ -1,12 +1,22 @@
...
 assignees: ''
 
 ---
 
-### Which content is outdated?
+### Location
 
-### What should be the new content?
+### Description
+Describe which content is outdated, and how it should be updated
```
Comment (chris529) [inline .github/ISSUE_TEMPLATE/update-content.md]:
```suggestion
Describe which content is outdated, and how it should be updated.
```

**Review by chris529 — COMMENTED** [review]
Looks good, just a few very minor points 👍

File: .github/ISSUE_TEMPLATE/improvement.md [hunk 8]
```diff
@@ -1,10 +1,25 @@
...
 ### Description
+Explain how the content should be improved.
+
+### Location
+
+#### Affected branch / URL
+Choose the applicable branch that should be used as the base branch for this issue.
+
+#### Page location
+Define the path and name for the new page in the documentation.
```
Comment (TalhaMaliktz) [inline .github/ISSUE_TEMPLATE/improvement.md]:
```suggestion
Insert the path of the affected page(s) in the documentation.
```

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]
Hey, nicely done. Just one small change as discussed. :)

**Review by TalhaMaliktz — APPROVED** [review]

**Pull request merged**